linux_os/guide/system/auditing/configure_auditd_data_retention/var_auditd_disk_full_action.var

```diff
@@ -10,8 +10,9 @@ interactive: false
 
 options:
     default: single
-    email: email
+
```

Comment:
Why is email removed? Also added empty space here.

Reply:
> Why is email removed? Also added empty space here.

Sorry for white spaces, I will fix them all. Email is not mentioned in the man page so I removed it.